ansible/roles/software/ckan/catalog/ckan-app/templates/catalog-next/etc_ckan_production.ini.j2

```diff
@@ -196,6 +196,9 @@ set debug = False
 # ckanext-harvest will use ckan.redis.url if redis configuration
 # is not specified here.
 
+# Mark as finished Jobs in 'Running' status after x minutes  
```

Comment:
it would be great if the comment said "after 24 hours" so I don't have to do any math :-)

Reply:
👍  I like this comment, we can add the 24 hour hint inline below.